build/commands/lib/pylint.js

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+const path = require('path')
+const fs = require('fs')
+const config = require('../lib/config')
+const util = require('../lib/util')
+
+const getDefaultOptions = () => {
+  let options = Object.assign({}, config.defaultOptions)
+  options.cwd = path.resolve(__dirname, '../../..')
+  return options
+}
+
+const runPylint = (args, continueOnFail = false) => {
+  let cmd_options = getDefaultOptions()
+  cmd_options.continueOnFail = continueOnFail
+  if (process.platform !== 'win32') {
+    util.run('which', ['pylint'], cmd_options)
+  } else {
+    util.run('where', ['pylint'], cmd_options)
+  }
+  util.run('pylint', ['--version'], cmd_options)
+  util.run('pylint', args, cmd_options)
+}
+
+const runGit = (gitArgs) => {
+  // Have to use shell because otherwise git gets confused with node's escaping
+  // of the args.
+  let prog = util.run('git', gitArgs, { shell: true })
+  const output = prog.stdout.toString()
+  console.log(output)
+  return output.trim()
+}
+
+const getChangedFiles = (base_branch, check_folders) => {
+  const merge_base = runGit(['merge-base', 'origin/' + base_branch, 'HEAD'])
+  if (!merge_base) {
+    console.log('Could not determine merge-base.')
+    process.exit(1)
+  }
+  const changed_files = runGit(['diff', '--name-only', '--diff-filter',
+    'drt', merge_base, '--', check_folders.join('/*.py ') + '/*.py'])
+  if (!changed_files) {
+    return []
+  }
+  return changed_files.split('\n')
```

Comment:
Do we need to use [node's `EOL`](https://nodejs.org/api/os.html#os_os_eol) for compatability?

Reply:
Updated with `EOL` on line 81 above but here and in `getAllFiles` `EOL` didn't work for me on Windows.